plugin/evm/block_builder.go

```diff
@@ -80,15 +80,15 @@ type blockBuilder struct {
 	// isSC is a boolean indicating if SubnetEVM is activated. This prevents us from
```

Comment:
nit: `isSE`

Reply:
It seems like there are a few other places as well that aren't in the diff which haven't been updated. A quick search for SC in `plugin/evm/` should find all of them that need to be replaced.

Reply:
done